seed/landing/tests.py

```diff
@@ -31,25 +29,25 @@ def test_simple_login(self):
         self.client.post(self.login_url, self.user_details, secure=True)
         self.assertTrue('_auth_user_id' in self.client.session)
 
-    def test_tos_login(self):
-        """
-        Happy path login when there is a ToS.
-        """
-        tos = "Agree to these terms"
-        TermsOfService.objects.create(active=True,
-                                      content=tos)
-        res = self.client.post(self.login_url, self.user_details, secure=True)
-
-        # expect to see the tos, in a form with an action to confirm
-        self.assertContains(res, tos)
-        expected_action = 'action="%s"' % self.tos_url
-        self.assertContains(res, expected_action)
-
-        # django-tos does not log the user in yet
-        self.assertFalse('_auth_user_id' in self.client.session)
-
-        # and submitting 'accept=accept' to the confirm url logs you in
-        self.client.post(self.tos_url, {'accept': 'accept'})
-
-        # now we're logged in
-        self.assertTrue('_auth_user_id' in self.client.session)
+    # def test_tos_login(self):
```

Comment:
Any reason to keeping this commented code around?

Reply:
I kept it around to make sure it was explicit that we aren't handling TOS anymore... and haven't been for quite some time.